lib/common/models/wp_user/existable.rb

```diff
@@ -76,7 +76,10 @@ def self.display_name_from_body(body)
         # replace UTF chars like  &#187; with dummy character
         title_tag.gsub!(/&#(\d+);/, '|')
 
-        name = title_tag[%r{([^|«»]+) }, 1]
+        name = title_tag[%r{.+» (.+)}, 1]

```

Comment:
the problem with this chars is, they are theme dependant. If you use another theme, the chars may be missing or different.
Maybe we can improve the regex like search for word characters as the name instead of relying on the special chars. Or combine both methods